run/HEMCO_Config.rc.025x03125.GEOSFP.DustL23M

```diff
@@ -0,0 +1,145 @@
+#
+# !REVISION HISTORY:
+#  See https://github.com/geoschem/hemco for complete history
+#EOP
+#------------------------------------------------------------------------------
+#BOC
+###############################################################################
+### BEGIN SECTION SETTINGS
+###############################################################################
+#-----------------------------------------------------
+# Settings for HEMCO standalone input and output
+#-----------------------------------------------------
+ROOT:                /ExtData/HEMCO
+METDIR:              /ExtData/GEOS_0.25x0.3125/GEOS_FP
+GridFile:            HEMCO_sa_Grid.025x03125.rc
+SpecFile:            HEMCO_sa_Spec.rc
+TimeFile:            HEMCO_sa_Time.rc
+DiagnFile:           HEMCO_Diagn.rc
+Logfile:             *
+#-----------------------------------------------------
+# Settings for HEMCO grid
+#-----------------------------------------------------
+MET:                 GEOSFP
+RES:                 025x03125
+NC:                  nc
+#-----------------------------------------------------
+# Settings for HEMCO standalone diagnostics
+#-----------------------------------------------------
+DefaultDiagnOn:      false
+DefaultDiagnSname:   TOTAL_
+DefaultDiagnLname:   HEMCO_total_emissions_
+DefaultDiagnDim:     2
+DefaultDiagnUnit:    kgm-2s-1
+DiagnPrefix:         OutputDir/HEMCO_sa_diagnostics
+DiagnFreq:           00000000 010000
+#-----------------------------------------------------
+# Debugging options
+# (set verbose to true to toggle debug printout)
+#-----------------------------------------------------
+Unit tolerance:      1
+Negative values:     0
+Verbose:             false
+#-----------------------------------------------------
+# Additional settings
+#-----------------------------------------------------
+PBL dry deposition:  False
+
+### END SECTION SETTINGS ###
+
+###############################################################################
+### BEGIN SECTION EXTENSION SWITCHES
+###############################################################################
+### Set the below switches below to false and HEMCO extensions to off. They
+### are only needed for GEOS-Chem simualations, but not for HEMCO standalone
+### simulations. Turning off the extensions also avoids having to supply
+### certain data fields for the extentions that usually are obtained directly
+### from GEOS-Chem.
+###############################################################################
+# ExtNr ExtName                on/off Species
+0       Base                   : on   *
+# ----- MAIN SWITCHES ---------------------------------------------------------
+    --> EMISSIONS              :       true
+    --> METEOROLOGY            :       true   # 1980-2021
+    --> CHEMISTRY_INPUT        :       false
+125     DustL23M               : on    TDST/DSTbin1/DSTbin2/DSTbin3/DSTbin4/DSTbin5/DSTbin6/DSTbin7
```

Comment:
As @msulprizio pointed out in the companion GEOS-Chem PR, this extension number differs from that in GEOS-Chem.